portfolio/src/main/java/com/google/sps/data/BlogHashMap.java

```diff
@@ -24,7 +24,7 @@
 import java.util.Set;
 
 public final class BlogHashMap {
-  private final String[] INTERNAL_TAGS = {"#general", "#wellbeing", "#music", "#games", "Tags"};
+  private final String[] INTERNAL_TAGS = {"#general", "#animals", "#art", "#business", "#education", "#environment", "#games", "#music", "#politics", "#wellbeing", "Tags"};
```

Comment:
(1) line is too long and please break.

(2) I think we should define all these tags as constant variables.